packages/components/src/components/FormInput/FormInput.tsx

```diff
@@ -51,6 +51,7 @@ const FormInput = React.forwardRef<HTMLInputElement, FormInputProps>(
           value={value}
           {...props}
         />
+        {!hasError && !helpText && <Box.div h="1rem" marginTop="space20" />}
```

Comment:
Don't we just need to be looking at helpText and not both has error and help text?

Reply:
@RohitRajendran I think both on this one since you can have `errorText` without having `helpText`.

Reply:
error text would come in through the`helpText` prop though right? So If there's HelpText then we don't need the spacer.

Reply:
@RohitRajendran I added an `errorText` prop as well so teams can pass different strings.

Reply:
That errorText prop was in ControlledFormInput and not in the FormInput though. I see in ControlledFormInput that it still gets passed into FormInput through the helpText prop so you still wouldn't need to check `hasError` inside of this component, just `helpText`. Or were you planning on adding the errorText prop to FormInput and HelpText components too?

Reply:
I see what you're saying now. Yep. Just checking `helpText`. `errorText` shouldn't be passed to FormInput.